api/tests/Api/Login/LoginTest.php

```diff
@@ -122,7 +122,7 @@ public function testLoginWithEmail() {
             '/authentication_token',
```

Comment:
this is now the same test as testLoginWithValidCredentials

Reply:
Fixed with https://github.com/ecamp/ecamp3/pull/3004/commits/5e3414515ef3d4cf2f72024a4a19b7c33974c38d